admin/release-packaging/centos/development/clusterhq.repo

```diff
@@ -0,0 +1,5 @@
+[clusterhq]
```

Comment:
We shouldn't have both a development and a marketing `clusterhq-release` package. Instead, we should just have a single package that has two repositories defined, one enabled and the other disabled by default.